JSON and Zarr updates

govekk opened the pull request:
- JSON API uses ezomero, not json_session.py (same functions)
- Added verify=False to JSON API call to work with dev server (and notes about changing that for research server)
- Changed example zarr url to one that actually exists
- Loaded config.py in zarr for OMEROHOST, made variable in shell login
- Changed image, project, dataset, user IDs all around to current user-1 so that I could run the notebooks (each student will have to use their own IDs)
- Due to indexing errors, changed `s7[z,0,0,:,:]` to `s7[z,0,:,:]` in last Zarr.ipynb block
- Added template config file

govekk committed:
added mac environment, initial outline of readme

govekk committed:
changed jupyter notebook version in Dockerfile to python-3.8.8

govekk committed:
removed pip setup for now

govekk committed:
renaming windows yml

govekk committed:
combined to one environment.yml and specified python version

govekk committed:
Merge branch 'TheJacksonLaboratory:main' into main

govekk committed:
changed json to use ezomer, added verify=False, changed example zarr url, load config for zarr login, changed IDs for user-1 dev, fixed indexing error in last zarr block

govekk committed:
added config template

erickmartins commented:
Couple comments:

1. might be worth updating the dependencies in `environment.yml` - ezomero at 1.2.0, omero-py at whatever the newest version is, etc etc
2. I'd add that  `OMEROWEB` needs a schema (https or something) before the address in `config_template.py`
3. the very last `imshow` in the zarr notebook (the one inside the for loop) gives me a 
```Invalid shape (1, 240, 305) for image data``` 
error. Using 
```plt.imshow(s7[z,0,:,:].squeeze())``` seems to have solved it!

govekk committed:
removed pip install from zarr notebook

govekk committed:
http note in config template

govekk committed:
updated ezomero, omero-py, ome-zarr, zarr, omero-cli-zarr package versions

govekk commented:
1. Updated dependencies for ezomero, omero-py, ome-zarr, zarr, omero-cli-zarr (most were just .0 - .1 changes)
2. Added comment in config_template.py to include http/https for omeroweb
3. Removed pip install block from zarr notebook that was installing old version of zarr with outdated shape

erickmartins commented:
all looks good - merging it
